server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
```

Comment:
Yes, this has been taken care of , just need to change the typo for the file name .